dockerfiles/theia/e2e/src/docker-run.sh

```diff
@@ -6,13 +6,32 @@
 #
 # SPDX-License-Identifier: EPL-2.0
 
+# (5 + 2 * 30 = 65 seconds) is the default timeout.
+: ${WAIT_COUNT:=30}
```

Comment:
FYI there is warning on https://www.shellcheck.net/

Line 10:
: ${WAIT_COUNT:=30}
  ^-- SC2223: This default assignment may cause DoS due to globbing. Quote it.
 
Line 26:
    WAIT_COUNT=$(($WAIT_COUNT-1));
                  ^-- SC2004: $/${} is unnecessary on arithmetic variables.

Reply:
Hmm... At least `$(())` is needed, AFAIK...

```
$ sh -c 'a=3; a=$(($a-1)); echo $a'
2
$ sh -c 'a=3; a=$($a-1); echo $a'
sh: 1: 3-1: not found
$ sh -c 'a=3; a=(($a-1)); echo $a'
sh: 1: Syntax error: "(" unexpected
```

Reply:
```
 sh -c 'a=3; a=$((a-1)); echo $a'
```

(no $ sign before a)

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2004

Reply:
Thanks for your information. I see and pushed some fixes. I'm going to check my fixes by my CI server.